apps/smeargle/src/components/Header.tsx

```diff
@@ -0,0 +1,15 @@
+import logoLarge from '../../public/assets/logo-large.svg';
+import logoSmall from '../../public/assets/logo-small.svg';
+
+export default function Header() {
+  return (
+    <header class="border-b-1 dark:border-whinehouse border-snow fixed z-10 flex min-h-[80px] w-full items-center bg-white px-4 dark:bg-black sm:px-8 lg:px-16">
+      <img
+        class="hidden w-[155px] md:flex"
+        src={logoLarge}
+        alt="tonemato logo"
+      />
+      <img class="w-[33px] md:hidden" src={logoSmall} alt="tonemato logo" />
```

Comment:
I think the big Logo is fine for Mobile as well no?

![Large Logo on Mobile](https://user-images.githubusercontent.com/18482238/232324321-fd3ec1a3-01bc-439a-adfa-8ab8240484a9.png)

Reply:
Well it works right now but wait for the filter component to come :p don't want to put that in a burger menu or something